src/plugins/dashboard/public/application/embeddable/types.ts

```diff
@@ -25,5 +25,5 @@ export type SavedObjectId = string;
 export interface DashboardPanelState<
   TEmbeddableInput extends EmbeddableInput | SavedObjectEmbeddableInput = SavedObjectEmbeddableInput
 > extends PanelState<TEmbeddableInput> {
-  readonly gridData: GridData;
```

Comment:
I replaced the readonly modifier while reviewing the changes, and everything continued to work the same way. Is there any reason to remove this modifier? If not, I would suggest keeping it in place.